README.md

```diff
@@ -88,7 +93,7 @@ optional arguments:
   --save SAVE           Path to a csv file to save results to.
   --retry-count RETRY_COUNT
                         retry count upon failure (defaults to 2, one retry).
-  --timeout TIMEOUT     timeout (seconds) to provide to the requests library
+  --timeout TIMEOUT     timeout (minutes) to provide to the aiohttp library
```

Comment:
We should not the change the API here - if the user provides seconds, we should just convert to minutes for the library.

Reply:
That makes sense. I tried to keep the numbers the same, that's why I only changed the unit, but I will fix this.